core/variant/dictionary.cpp

```diff
@@ -33,6 +33,11 @@
 #include "core/templates/ordered_hash_map.h"
 #include "core/templates/safe_refcount.h"
 #include "core/variant/variant.h"
+// required in this order by VariantInternal, do not remove this comment.
```

Comment:
It's not too important, but if you end up having to amend to do other changes, I'd add an empty line before the comment to make a clear separation between the normally ordered includes and the special ones. Otherwise people will just add after `variant_internal.h` instead of `variant.h`.